staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -32,17 +32,19 @@ import (
 	"k8s.io/apiserver/pkg/audit"
 	"k8s.io/apiserver/pkg/endpoints/handlers/negotiation"
 	"k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/features"
 	"k8s.io/apiserver/pkg/registry/rest"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	utiltrace "k8s.io/apiserver/pkg/util/trace"
 )
 
-func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Interface, includeName bool) http.HandlerFunc {
+func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Interface, includeName bool, gates utilfeature.FeatureGate) http.HandlerFunc {
```

Comment:
I think this should take a struct that contains the pieces it cares about.  So in this case, a struct that contains `DryRun bool`.

```go
type RESTHandlerFeatures struct{
    DryRun bool
}
```